AirCastingTests/SearchAndFollowTests/AirBeamMeasurementDownloaderTests.swift

```diff
@@ -0,0 +1,49 @@
+// Created by Lunar on 03/08/2022.
+//
+import XCTest
+import Resolver
+@testable import AirCasting
+
+class AirBeamMeasurementDownloaderTests: ACTestCase {
```

Comment:
I think it would be great to add some more tests here:
1. checking if completion is called correctly
2. checking for query items (1440 measurements limit)
3. perhaps we can mock a response from server and check decoding (if snake case is decoded correctly etc.)

Reply:
I am going to add one, but as this PR got enough big and I am near holiday time - I going to push it more onto merge state then new thing state. With all the pleasure, I can add more tests in the future 🫡